apps/dashboard/lib/auth/workos.ts

```diff
@@ -746,6 +792,24 @@ export class WorkOSAuthProvider extends BaseAuthProvider {
               sameSite: "lax",
             },
           },
+          {
+            name: UNKEY_ACCESS_TOKEN,
+            value: accessToken,
+            options: {
+              secure: true,
+              httpOnly: true,
+              sameSite: "lax",
```

Comment:
A minor question... 

Do we think using lax is fine here and not `strict`? Requests should be sent and received from app.unkey.com right?